imports/plugins/included/product-variant/register.js

```diff
@@ -8,7 +8,7 @@ Reaction.registerPackage({
   registry: [{
     route: "/tag/:slug?",
     name: "tag",
-    template: "products",
+    template: "Products",
```

Comment:
I believe this needs a migration since this goes into the database. See migration 27 for an example